search/includes/classes/class-search.php

```diff
@@ -314,4 +317,17 @@ public function filter__jetpack_widgets_to_include( $widgets ) {
 		// Flatten the array back down now that may have removed values from the middle (to keep indexes correct)
 		return array_values( $widgets );
 	}
+
+	/*
+	 * Add track track_total_hits to formatted_args
```

Comment:
Minor / non-blocking - I know we'll end up using this filter for other stuff, so maybe we can make the comment more generic to start with?